server.js

```diff
@@ -20,13 +18,223 @@ mongoose.Promise = Promise;
+  } else {
+    res.status(503).json({error: 'Service unavilable'})
+  }
+})
+
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send(
+    {"Welcome":"This is an open API with Hammarbys football players 2022.",
+    "Database": "MongoDB",
+    "Endpoints": "/endpoints",
+    "Routes":[{
+    "/hammarby":"Get all the players",
+    "/hammarby/names/:name":"Get a player by name",
+    "/hammarby/ages/:age":"Get the player with a specific age",
+    "/hammarby/positions/:position":"Get player based on their position (eg: striker, midfielder, defender, goalkeeper)",
+    "/hammarby/born/:year":"Get player based on the year they were born (eg: 1991)",
+    "/hammarby/nationalitys/:nationality":"Get player based on their nationality (eg: Sweden, Denmark, Albania, Gambia)",
+    "/hammarby/goals/:goal":"Shows the players who has done at least one goal for Hammarby (yes / no)",
+    "/hammarby/numbers/:number":"Shows the player with a specific shirt number (eg: 15)"
+  }],
+  "Querys":[{
+    "/hammarby/players?shirt_number=number":"Shows the player with a specific shirt number (eg: 15)",
+    "/hammarby/players?name=name":"Shows the named player (eg: Gustav Ludwigson)",
+    "/hammarby/players?position=position":"Shows player based on their position (eg: striker, midfielder, defender, goalkeeper)",
+    "/hammarby/players?age=age":"Get the player with a specific age (eg: 18)",
+    "/hammarby/players?year_born=year":"Shows players that was bort on a specific year (eg: 1991)",
+    "/hammarby/players?nationality=nationality":"Shows players based on their nationality (eg: Sweden, Denmark, Albania, Gambia)",
+    "/hammarby/players?done_goal=goal":"Shows the players who has done at least one goal for Hammarby (yes / no",
+
+    "You can play around with these endpoitns and querys":"Eg: /hammarby?born=1991&goal=yes",
+    }]
+}
+  )
+});
+
+app.get('/endpoints', (req, res) => {
+  res.send(listEndpoints(app));
+})
+
+// Get all players
+app.get('/hammarby', async (req, res) => {
+  const players = await Player.find()
+  res.json(players)
+})
+
+// Find players by position
+app.get('/hammarby/positions/:position', async (req, res) => {
+  try {
+    const playerPosition = await Player.find({ position: req.params.position})
+    if (playerPosition.length === 0) {
+      res.status(404).json({error: 'position not found'})
+    } else {
+      res.json(playerPosition)
+    }
+  } catch (err) {
+    res.status(400).json({ error: 'Invalid position'})
+  }
+})
+
+// Find players by age
+app.get('/hammarby/ages/:age', async (req, res) => {
+  try {
+    const playerAge = await Player.find({ age: req.params.age})
+    if (playerAge.length === 0) {
+      res.status(404).json({error: 'Age not found'})
+    } else {
+      res.json(playerAge)
+    }
+  } catch (err) {
+    res.status(400).json({ error: 'Invalid age'})
+  }
+})
+
+// Find players by name
```

Comment:
maybe you can try less case sensitive query for names by using .toLowerCase() or .toUpperCase()?